orion-ui/tests/workQueues.spec.ts

```diff
@@ -0,0 +1,89 @@
+import { expect } from '@playwright/test'
+import { mocker } from '@prefecthq/orion-design'
+import { test, useForm, useLabel, useTable, usePageHeading, useLink, pages, useIconButtonMenu, useModal, useButton } from './utilities'
+import { useToggle } from './utilities/useToggle'
+
+test.describe.configure({ mode: 'serial' })
+
+test.beforeEach(async ({ page }) => {
+  await page.goto(pages.workQueues)
+})
+
+const workQueueNameToCreate = mocker.create('string')
+
+test('Can create work queue', async ({ page }) => {
+  const { table, rows: workQueues } = useTable()
+  const existingWorkQueues = await workQueues.count()
+
+  await createWorkQueue()
+
+  await page.goto(pages.workQueues)
+
+  await table.waitFor()
+  const newWorkQueues = await workQueues.count()
+
+  expect(newWorkQueues).toBe(existingWorkQueues + 1)
+})
+
+test('Can toggle workQueue from list', async () => {
+  const { rows: worksQueues } = useTable()
+  const worksQueue = worksQueues.filter({
+    hasText: workQueueNameToCreate,
+  })
```

Comment:
If you accept name in your createWorkQueue function. Could we have each test create its own work queue and run these in parallel?

Would have to tweak how we're testing that something is created. We'd have to locate a table row that contains the unique name rather than checking the count is 1 greater. But I think that's better right?